src/image-labels-options.ts

```diff
@@ -1,6 +1,51 @@
+/**
+ * Options for the image labels plugin.
+ */
 export interface ImageLabelsOptions {
+    /**
+     * Whether or not to display images.
+     * @default true
+     */
     display?: boolean;
+    /**
+     * Maximum size of the image. Will get shrinked if space is not enough.
+     * @default 48
+     */
     imageSize?: number;
+    /**
+     * The padding closer to the chart.
+     *
+     * Top padding for horizontal direction.
+     * Right padding for vertical direction.
+     * @default 10
+     */
+    paddingClose?: number;
+    /**
+     * The padding further from the chart.
+     *
+     * Bottom padding for horizontal direction.
+     * Left padding for vertical direction.
+     * @default 0
+     */
+    paddingFar?: number;
+    /**
+     * The list of images to display above each category axis label. Null values are skipped.
```

Comment:
You should mention that for null values we just use the placeholder image, we're not actually skipping them.